lib/ansible/modules/cloud/vmware/vmware_vm_vm_drs_rule.py

```diff
@@ -46,17 +46,20 @@
     - If set to C(True), the DRS rule will be enabled.
     - Effective only if C(state) is set to C(present).
     default: False
+    type: bool
```

Comment:
Changes to files (this and others) unrelated to the new module should be in a separate PR.